### DISCUSSION TITLE
PKCS#12 String Format

### DISCUSSION THREAD

**Discussion by joevandeventer** [jerson/flutter-rsa#64] [Q&A] [answered]
Hi! I'm working on implementing this in my Flutter app, and had a question - I'm trying to import a PKCS#12 file, which is a binary format, but the library specifically requests a string. Should I Base64 encode the binary data?

**Comment by jerson** [comment, marked as answer]
Hey @joevandeventer yeah it should be base64

**Reply by jerson** [reply]
the reason behind that is because it was originally created without consider binary support, but I think this is something we can improve to support binary instead

**Reply by joevandeventer** [reply]
Thanks for getting back to me so fast! That's a huge help - at least I know where I'm _not_ screwing up at the moment. :)
